recipes/libffi/all/conanfile.py

```diff
@@ -1,20 +1,25 @@
-from conan.tools.microsoft import msvc_runtime_flag
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-import contextlib
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.env import Environment, VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, mkdir, replace_in_file, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc, msvc_runtime_flag, unix_path
+from conan.tools.scm import Version
+import glob
 import os
 import shutil
 
-required_conan_version = ">=1.36"
+required_conan_version = ">=1.52.0"
 
 
-class LibffiConan(ConanFile):
+class PackageConan(ConanFile):
```

Comment:
Not that it matters...

```suggestion
class LibffiConan(ConanFile):
```

Reply:
I hadn't even noticed that was changed. Thanks. Fixed.